src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(({ id }) => activeTabId === id) || tabs[0];
```

Comment:
You really don't need to create absolutely the same (like `activeTab`).
Just relay `activeTab` to <Tabs />